README.md

```diff
@@ -1,11 +1,15 @@
 # Check eligibility for Great British Insulation Scheme
 
 Formerly known as "Help to Heat".
-
 ## Initial Setup
-
-1. From the project root, run `cp envs/web.template envs/web`
-2. Populate the OS API key into the OS_API_KEY variable in `envs/web`. The key can be found in Keeper. Please only use a single value (i.e. `["my_key_here"]`), rather than all of the keys that are stored in Keeper, in order to avoid needing to rotate all of the keys if your local environment is accidentally leaked.
+### (Optional) Poetry Setup 
+If you want to be able to make changes to the dependency list using Poetry, or install the dependencies locally (so you can click into them in your IDE, or use autocomplete) perform these steps:
+1. [Install Poetry](https://python-poetry.org/docs/) on your machine
+2. [Install Python 3.9.13](https://www.python.org/downloads/release/python-3913/)
+3. Set up a virtual environment within the project for Poetry to manage your dependencies. A guide for Pycharm can be found [here](https://www.jetbrains.com/help/pycharm/poetry.html).
+### Main steps
+4. From the project root, run `cp envs/web.template envs/web`
+5. Populate the OS API key into the OS_API_KEY variable in `envs/web`. The key can be found in Keeper. Please only use a single value (i.e. `["my_key_here"]`), rather than all of the keys that are stored in Keeper, in order to avoid needing to rotate all of the keys if your local environment is accidentally leaked.
```

Comment:
Tiny thing now: can these steps move up under Initial Setup where they were before and the optional section be below them separately? I think it's easier to miss them this way round.

Reply:
That's true, I've swapped them around now :)